sydent/replication/pusher.py

```diff
@@ -128,10 +128,13 @@ def scheduledPush(self):
                 logger.debug("%d updates to push to %s", total_updates, p.servername)
                 if total_updates:
                     logger.info("Pushing %d updates to %s:%d", total_updates, p.servername, p.port)
-                    updateDeferred = p.pushUpdates(push_data)
-                    updateDeferred.addCallback(self._pushSucceeded, peer=p, ids=ids)
-                    updateDeferred.addErrback(self._pushFailed, peer=p)
-                    break
+                    try:
+                        updateDeferred = p.pushUpdates(push_data)
+                        updateDeferred.addCallback(self._pushSucceeded, peer=p, ids=ids)
+                        updateDeferred.addErrback(self._pushFailed, peer=p)
+                        break
```

Comment:
Why are we breaking here if we want to talk to all peers?

Reply:
I don't know actually, but somehow we manage to iterate to all of them despite this?